cmd/relayer/start/create_ibc_channel.go

```diff
@@ -8,11 +8,15 @@ import (
 	"path/filepath"
 )
 
+func getLogFilePath(home string) string {
```

Comment:
here you created a function for getting the path in the  da-light-client `start.go` line 25 you simply created the path inline.

Reply:
not the same path